samples/unit-tests/series/marker-clusters/demo.js

```diff
@@ -359,6 +359,74 @@ QUnit.test('General marker-clusters', function (assert) {
         resultVisibility,
         'Parent point should be hidden before animation.'
     );
+
+    // (#13302)
+    var done = assert.async(),
+        url = location.host.substr(0, 12) === 'localhost:98' ?
```

Comment:
I think it's a bad idea - for example I'm using only `utils.highcharts.com` so this won't work in browser.

Could you use for example `https://www.highcharts.com/samples/graphics/sun.png` ?

Reply:
I've tried `https://www.highcharts.com/samples/graphics/sun.png` yet it doesn't work in tests. I found this approach in other tests, and for me, it works fine in `utils.highcharts.com`.

Reply:
Okay then! 👍